flexget/plugins/output/send_telegram.py

```diff
@@ -0,0 +1,340 @@
+from __future__ import unicode_literals, division, absolute_import
+
+from sqlalchemy import Column, Integer, String
+
+from flexget import db_schema, plugin
+from flexget.event import event
+
+import telegram
```

Comment:
In the same vein as my last comment: Import `telegram` in a try-catch block inside `on_task_start` or `on_task_exit` -- not sure which is cleaner.